spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/session/JdbcSessionProperties.java

```diff
@@ -42,7 +43,10 @@
 	 */
 	private String tableName = DEFAULT_TABLE_NAME;
 
-	private final Initializer initializer = new Initializer();
+	/**
+	 * Create the required Spring Session tables on startup.
```

Comment:
Hmm, not sure what exactly you refer to by `removes silently a feature`, but that part is basically alignment with what you did with Quartz JDBC properties.